cli/src/main/java/org/jboss/as/cli/handlers/BaseOperationCommand.java

```diff
@@ -260,11 +269,16 @@ protected void addHeaders(CommandContext ctx, ModelNode request) throws CommandF
         opHeaders.set(headersNode);
     }
 
-    protected void handleResponse(CommandContext ctx, ModelNode response, boolean composite) throws CommandLineException {
+    protected void handleResponse(CommandContext ctx, OperationResponse response, boolean composite) throws CommandLineException {
         displayResponseHeaders(ctx, response);
+        handleAttachedFile(ctx, response);
+    }
+
+    protected void handleAttachedFile(CommandContext ctx, OperationResponse operationResponse) throws CommandLineException {
     }
 
-    protected void displayResponseHeaders(CommandContext ctx, ModelNode response) {
+    protected void displayResponseHeaders(CommandContext ctx, OperationResponse operationResponse) {
```

Comment:
Is it necessary to change this signature? This is somewhat a public API, as external extensions can write handlers, so we shouldn't change API if it's not required. I don't think any impl of this would want to do anything with the operationResponse besides call getResponseNode().